test.py

```diff
@@ -5,10 +5,8 @@
 
 from deeputil import *
 
-#suite = doctest.DocTestSuite(misc)
-
 def suite_maker():
-    suite= unittest.TestSuite()
+    suite = unittest.TestSuite()
```

Comment:
@shivam-deepcompute  modules not mentioned i.e. streaming_counter & timer

Reply:
@supriyopaul  please click on the expand button. There you will find them.